tools/generate_python_client.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
+
+set -o nounset
+
+temp_swagger_file=$(mktemp)
+sed '/pattern:/d' ${SWAGGER_FILE} > ${temp_swagger_file}
+temp_config_file=$(mktemp)
+echo '{"packageName" : "assisted_service_client", "packageVersion": "1.0.0"}' > ${temp_config_file}
```

Comment:
version is a bit disturbing but we can live with it for now.
Consider passing short git hash instead of hard coded 1.0.0

Reply:
Already answered @filanov 
I wanted to use git describe to get the current tag, but the current repository don't have tags at all